cli/scripts/bin/boomi_runtime_installer.sh

```diff
@@ -119,11 +116,13 @@ echo "install boomi runtime as $USR"
 cd /home/$USR/boomi/boomicicd/boomiinstall-cli/cli/scripts
 if [ -n "$efsMount" ] ; then
     echo "setting EFS Mount:${efsMount} ..."
-    source bin/efsMount.sh efsMount=${efsMount}
+    source bin/efsMount.sh efsMount=${efsMount} defaultAWSRegion=${EC2_REGION} platform=${platform}
```

Comment:
can we do without having to provide was region ? is it possible to preconstruct the mount share before passing to the script?